samples/line-time-point-data.html

```diff
@@ -3,9 +3,9 @@
 
 <head>
     <title>Time Scale Point Data</title>
-    <script src="../node_modules/moment/min/moment.min.js"></script>
+    <script src="../bower_components/moment/min/moment.min.js"></script>
```

Comment:
Wouldn't this make the example not work if someone runs `npm install`?  In my uses, I occasionally use bower to import production libs, but whenever I'm contributing to work, it's always the same `npm install` and everything is setup.  Thoughts?